src/consensus/vote.cpp

```diff
@@ -163,15 +158,15 @@ void VoteManager::addUnverifiedVotes(std::vector<Vote> const& votes) {
   }
 }
 
-void VoteManager::removeUnverifiedVote(uint64_t const& pbft_round, vote_hash_t const& vote_hash) {
+void VoteManager::removeUnverifiedVote(uint64_t const pbft_round, vote_hash_t const& vote_hash) {
```

Comment:
this makes a copy of uint64_t so const does not make any sense here

Reply:
Fixed, thanks